src/main/java/mate/controller/driver/LoginDriverController.java

```diff
@@ -0,0 +1,41 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.exception.AuthenticationException;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.AuthenticationService;
+
+public class LoginDriverController extends HttpServlet {
+    private static final String SESSION_ATTRIBUTE_ID = "driver_id";
```

Comment:
```suggestion
    private static final String SESSION_ATTRIBUTE = "driver_id";
```

lets make our code more consistent, in `GetMyCurrentCarsController` you named this attribute `SESSION_ATTRIBUTE`